tidb-server/main.go

```diff
@@ -323,6 +323,93 @@ func validateConfig() {
 		log.Error("TiDB run with skip-grant-table need root privilege.")
 		os.Exit(-1)
 	}
+	if cfg.Port < config.MinPortNumber || cfg.Port > config.MaxPortNumber {
+		log.Error("Invalid server port number.")
```

Comment:
We should also output the MinPortNumber/MaxPortNumber and cfg.Port in the error message.